index.html

```diff
@@ -109,8 +133,20 @@
           $("#jsonInput").val(JSON.stringify(testDataClinical, null, '    '));
         }
         $("#parseJson").on("click", function() {
-          allData = JSON.parse($("#jsonInput").val())
-          var timeline = clinicalTimeline
+          //initialise with advanced view
+          handleViews(true);
+        });
+        if (!waitForParse) {
+          $("#parseJson").click();
+        }
+      }).error(function() {
+          alert("Error loading JSON file.");
+        });
+    });
+
+    function handleViews(enableAdvanced) {
```

Comment:
I prefer `initViews` over `handleViews` here because it's initializing the state of the view buttons and the timeline